### DISCUSSION TITLE
requests.exceptions.ChunkedEncodingError: ('Connection broken: IncompleteRead(4001 bytes read, 35099 more expected)', IncompleteRead(4001 bytes read, 35099 more expected))

### DISCUSSION THREAD

**Discussion by korolenkowork** [celery/celery#9136] [Q&A]
Some of my jobs fail with requests.exceptions.ChunkedEncodingError

Why is that happening?

**Comment by thedrow** [comment]
There's no information to resolve your issue and it is likely that this is your error and not Celery's
